packages/remix-edge-adapter/src/index.ts

```diff
@@ -0,0 +1 @@
+console.log('nothing for now')
```

Comment:
I've stubbed each package out for now so that the lint rules don't fail. Without this you get an `exit 2` error because it found no files in the `packages` folder.